packages/protocol/test/stability/exchange.ts

```diff
@@ -376,13 +387,549 @@ contract('Exchange', (accounts: string[]) => {
     })
   })
 
-  describe('#exchange', () => {
+  describe('#getSellTokenAmount', () => {
+    it('should return the correct amount of sellToken', async () => {
+      const amount = 10
+      const sellAmount = await exchange.getSellTokenAmount(amount, true)
+
+      const expectedSellAmount = getSellTokenAmount(
+        new BigNumber(amount),
+        initialGoldBucket,
+        initialStableBucket
+      )
+
+      assert.equal(sellAmount.toString(), expectedSellAmount.toString())
+    })
+  })
+
+  // Run the following test for both these functions
+  const sellFunctionNames = ['sell', 'exchange']
+
+  for (const fnName of sellFunctionNames) {
+    describe(`#${fnName}`, () => {
```

Comment:
Style nit: not as important here, but still wold prefer `fnName` -> `sellFunctionName`.